UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -215,6 +216,20 @@ ScriptsTool::~ScriptsTool()
 		       ui->scripts->currentRow());
 }
 
+void ScriptsTool::updatePythonVersionLabel()
+{
+	QString label;
+	if (obs_scripting_python_loaded()) {
+		char version[8];
+		obs_scripting_python_version(version, sizeof(version));
```

Comment:
I wasn't sure if this was right at first, but afaik in C passing the name of the array implicitly passes a pointer to the first element of the array, so it should be correct.